src/install.js

```diff
@@ -18,11 +18,14 @@ function ensureCallback (plugin) {
   // Otherwise wrap synchronous plugins to match the async flow
   return function (app, options, next) {
     plugin.register(app, options)
+
     next(null)
   }
 }
 
 function installPlugin (next, plugin) {
+  if (!plugin.register) return next
```

Comment:
should this be considered an error state? is there a case where someone would want to add a plugin with no register function?